lib/jasmine-reportportal-reporter.js

```diff
@@ -34,17 +37,28 @@ class ReportportalReporter {
         return 'TEST';
     }
 
+    setAttributes(attributes) {
+        this.additionalCustomParams = { ...this.additionalCustomParams, ...attributes };
```

Comment:
Please use `Object.assign` here too to be able to support wider range of node versions.